pkg/common/ipBlock.go

```diff
@@ -186,32 +186,44 @@ func addIntervalToList(ipbNew *IPBlock, ipbList []*IPBlock) []*IPBlock {
 	return ipbList
 }
 
-func NewIPBlockFromCidr(cidr string) *IPBlock {
+// NewIPBlockFromCidr returns a new IPBlock object from input CIDR string
+func NewIPBlockFromCidr(cidr string) (*IPBlock, error) {
 	res, err := NewIPBlock(cidr, []string{})
 	if err != nil {
-		return nil
+		return nil, err
 	}
-	return res
+	return res, nil
+}
+
+// PairCIDRsToIPBlocks returns two IPBlock objects from two input CIDR strings
+func PairCIDRsToIPBlocks(cidr1, cidr2 string) (ipb1, ipb2 *IPBlock, err error) {
```

Comment:
This function is called twice from `ibmVPC` - wouldn't it make more sense to put it there?

Reply:
I think it is general functionality for `IPBlock`, might be used by other packages in the future..